src/lib.rs

```diff
@@ -38,6 +38,7 @@
     html_logo_url = "https://raw.githubusercontent.com/smol-rs/smol/master/assets/images/logo_fullsize_transparent.png"
 )]
 #![cfg_attr(docsrs, feature(doc_auto_cfg))]
+#![feature(doc_cfg)]
```

Comment:
```suggestion
#![cfg_attr(docsrs, feature(doc_cfg, doc_auto_cfg))]
```

Reply:
I didn't think it could be used together. It has been modified.